pro/src/ui-kit/Banners/BannerLayout/BannerLayout.tsx

```diff
@@ -45,27 +46,30 @@ const BannerLayout = ({
       <span className={styles['border-cut']}>
         {closable && (
           <button onClick={handleOnClick} type="button">
-            {isNewStyles &&
-            type != 'new' &&
-            type != 'light' &&
-            type != 'image' ? (
-              <IcoClearIcon
-                title="Masquer le bandeau"
-                className={cn(styles['close-icon-banner'])}
-              />
-            ) : (
-              <CloseIcon
-                title="Masquer le bandeau"
-                className={styles['close-icon']}
-              />
-            )}
+            {
+              /* istanbul ignore next: DEBT, TO FIX */
+              isNewStyles && type != 'new' && type != 'light' ? (
+                <IcoClearIcon
+                  title="Masquer le bandeau"
+                  className={cn(styles['close-icon-banner'])}
+                />
+              ) : (
+                <CloseIcon
+                  title="Masquer le bandeau"
+                  className={styles['close-icon']}
+                />
+              )
+            }
           </button>
         )}
 
         <div className={styles['content']}>
-          {type === 'new' && (
-            <Icon svg="ico-star" className={styles['ico-new']} />
-          )}
+          {
+            /* istanbul ignore next: DEBT, TO FIX */
```

Comment:
je pense pas que ce soit des DEBT, TO FIX mais peut-être des `/* istanbul ignore next: graphic variation */` ?

Reply:
Effectivement. Corrigé.